.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,173 @@
+##########################################################################
+#
+# WARNING: This file was generated by update.php.
+#
+# You can find the relevant template in the `/templates` folder.
+#
+
+name: Build and publish images to Docker Hub
+
+on:
+  push:
+    branches: master
+  # Once weekly On Sundays at 00:00 UTC.
+  schedule:
+    - cron: '0 0 * * 0'
+
+env:
+  REGISTRY_USERNAME: garypendergast
+  REGISTRY_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  PACKAGE_REGISTRY_HOST:
+  PACKAGE_REGISTRY: wordpressdevelop
+  PR_TAG:
+
+jobs:
+  # Checks for any incomplete preceeding workflow runs and cancels them.
+  check-for-duplicates:
+    name: Cancel duplicate workflow runs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+  build-php-images:
+    name: PHP ${{ matrix.php }} Image
+    runs-on: ubuntu-latest
+    needs: check-for-duplicates
+    strategy:
+      matrix:
+        php: [ '5.2', '5.3', '5.4', '5.5', '5.6.20', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+    env:
+      PHP_VERSION: ${{ matrix.php }}
+      PHP_LATEST: '7.4'
```

Comment:
Not 8.0? 😀

Reply:
I'm going to make that change in a separate PR after this one is merged and builds are back working correctly. For now, this copies the "latest" value currently being used.